.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
Is the .vscode folder needed here? Maybe we should add this to .gitignore

Reply:
Not really. Yeah, sure.